aikau/src/main/resources/alfresco/upload/templates/UploadMonitor.html

```diff
@@ -0,0 +1,15 @@
+<div class="${baseClass}">
+   <table class="${baseClass}__items">
+      <thead>
+         <tr class="${baseClass}__item-heading">
+            <td class="${baseClass}__item__name">&nbsp;</td>
+            <td class="${baseClass}__item__progress">&nbsp;</td>
+            <td class="${baseClass}__item__status">&nbsp;</td>
+            <!-- <td class="${baseClass}__item__close-button">&nbsp;</td> -->
```

Comment:
This has presumably been left commented out for later addition in future sprint?